package.json

```diff
@@ -5,15 +5,18 @@
   "main": "src/js/index.js",
   "scripts": {
     "commitlint": "commitlint --from 5ed6165",
-    "test": "check-node-version --npm 8.5.5 && nyc mocha -- -R spec 'tests/**/*.test.js'",
+    "check-node-version": "check-node-version --node $(cat .nvmrc) --npm 10.5.0 --print",
+    "test": "nyc mocha -- -R spec 'tests/**/*.test.js'",
```

Comment:
I don't see the `-R` "reporter" option configured in many other projects. I know it was here before, but should it continue to be included?